battleship-client/src/main/java/com/academy/solid/nie/client/ui/ShipPlacer.java

```diff
@@ -86,19 +83,20 @@ private Type shipOrientation(MouseEvent event) {
         return points;
     }
 
-    void placeShipsOfEnemy(String shipsString) {
+    private void placeShipsOfEnemy(String shipsString) {
         String[] ships = shipsString.split(",\\|");
```

Comment:
Are these splitting signs described somewhere ? It's kind of hard to figure out what they mean